### PR TITLE
UX pass on APV settings window

### PR DESCRIPTION
**Pull request by adrien-de-tocqueville**
### Purpose of this PR

first UX pass on APV window

* rename baking states to lighting scenarios
* move the table up in the UI
* add foldouts for sections
* add stats section
* add active scenario label
* clicking on the left part of "generate lighting" don't open the popup anymore but does the same as "bake the set"

Probably some variables are still named state instead of scenario but it's only local variables, no big deal

Now
image

Before
image

---
### Testing status

Tested the UI and and stats panel. Most of the changes are renaming

### CODE REVIEW

File: com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeBakingProcessSettingsDrawer.cs [hunk 1]
```diff
@@ -1,9 +1,10 @@
 using UnityEditor;
+using UnityEditor.Rendering;
 
 namespace UnityEngine.Rendering
 {
     [CustomPropertyDrawer(typeof(ProbeVolumeBakingProcessSettings))]
-    class ProbeVolumeBakingProcessSettingsDrawer : PropertyDrawer
+    internal class ProbeVolumeBakingProcessSettingsDrawer : PropertyDrawer
```
Comment (FrancescoC-unity) [inline com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeBakingProcessSettingsDrawer.cs:7]: Why changing to internal?

File: com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeBakingWindow.cs [hunk 2]
```diff
@@ -38,20 +38,26 @@ static class Styles
...
+            public static readonly GUIContent activeScenarioLabel = new GUIContent("Active Scenario", EditorGUIUtility.IconContent("FilterSelectedOnly").image);
             public static readonly GUIContent sceneNotFound = new GUIContent("Scene Not Found!", Styles.sceneIcon);
             public static readonly GUIContent bakingSetsTitle = new GUIContent("Baking Sets");
-            public static readonly GUIContent bakingStatesTitle = new GUIContent("Baking States");
             public static readonly GUIContent debugButton = new GUIContent(Styles.debugIcon);
+            public static readonly GUIContent probeVolumeProfile = new GUIContent("Probe Volume Profile");
+            public static readonly GUIContent stats = new GUIContent("Stats");
+            public static readonly GUIContent scenarioCostStat = new GUIContent("Active Scenario Size On Disk", "Size on disk used by the baked data of the currently selected lighting scenario.");
+            public static readonly GUIContent totalCostStat = new GUIContent("Baking Set Total Memory Cost", "Size on disk used by baked data of all lighting scenarios of the set.");
```
Comment (FrancescoC-unity) [inline com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeBakingWindow.cs:48]: I would use consistency here and call this "Baking Set Total Size On Disk"

File: com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeBakingProcessSettingsDrawer.cs (hunk 1 above)
Comment (adrien-de-tocqueville) [reply com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeBakingProcessSettingsDrawer.cs:7]: hum it's a leftover, but internal is the default accessibility for a class anyway

**Review by FrancescoC-unity — APPROVED** [review]
Standalone passes and given that it is just renaming/cosmetic it is safe to merge that way.

**Pull request merged**